pyproject.toml

```diff
@@ -196,6 +196,9 @@ requires-python = '>=3.9'
 'core.arithmetic.add_multiply' = 'aiida.workflows.arithmetic.add_multiply:add_multiply'
 'core.arithmetic.multiply_add' = 'aiida.workflows.arithmetic.multiply_add:MultiplyAddWorkChain'
 
+[project.entry-points.'aiida.orm']
+'core.code.abstract' = 'aiida.orm.nodes.data.code.abstract:AbstractCode'
```

Comment:
I think this entry point should be moved to [project.entry-points.'aiida.data'] section where the other code entry points are defined.